src/beautifiers/phpcbf.coffee

```diff
@@ -11,12 +11,10 @@ module.exports = class PHPCBF extends Beautifier
   isPreInstalled: false
 
   options: {
-    _:
-      standard: ["standard", (standard) ->
-        if (standard) then \
-          standard else "PEAR"
-      ]
-    PHP: true
+    PHP:
+      phpcbf_path: true
+      phpcbf_version: true
+      standard: true
```

Comment:
I think `standard` should still be the transform function from before? i.e `standard: ["standard", (standard) ->`..